data/mil2525d/utilities/style-utilities/merge-stylx-utilities/Readme.md

```diff
@@ -3,38 +3,53 @@
 
 ## Instructions 
 
-* Copy required files mentioned in Requirements above:
-    * `mil2525d-points-only.dat` 
-    * `mil2525d-lines-areas-labels-base-template.stylx` 
-    * `Military-All-Icons.csv`
-    * to this folder (`military-features-pro-data\source\utilities\merge-stylx-utilities`)
-* Rename  `mil2525d-lines-areas-labels-base-template.stylx` to `mil2525d.stylx`
-* Edit the versions.csv file to update the version information that will be inserted into the completed stylx file
-* Launch the Sqlite editor/interpreter of your choice
-* Follow the SQL commands/steps in [SqliteMergeStylx.sql](./SqliteMergeStylx.sql)
-    * Optionally you may edit this file to update the setting/string `{Full Path To}` to the local path and run this as an SQL script
-* Exit the Sqlite Editor
-* Test and update the resulting `mil2525d.stylx` in this repository
+* Copy/edit/verify the prerequisite files are present
+    * Copy the required "Points/Icons-Only" version of the .stylx mentioned in Requirements above to this folder and ensure it is named: `mil2525d-points-only.stylx`
+        * This file should now exist locally: `...utilities\style-utilities\merge-stylx-utilities\mil2525d-points-only.stylx`
+    * Clone the [joint-military-symbology-xml](https://github.com/Esri/joint-military-symbology-xml) repository locally if you have not already done so.
+        * This file should now exist locally: `...joint-military-symbology-xml\samples\imagefile_name_category_tags\Military-All-Icons.csv`
+    * Edit the versions.csv file to update the pertinent version information, this information will be inserted into the completed/merged mil2525d.stylx file
+        * This file should now be current with the updated/desired version information: `...utilities\style-utilities\merge-stylx-utilities\versions.csv`
+* Edit the automated scripts to point to the paths/configuration of your local machine:
+    * Edit the batch file: [AutomatedCopyandMerge.bat](./AutomatedCopyandMerge.bat), set/replace the following settings:
+        * These settings read `{TODO_EDIT_THIS_PATH}` and are:
+        * The local/cloned copy of the Joint Military Symbology repository: `SET LOCAL_JMSXML_REPO_CLONE=________`
+        * The path to the SQLite executable/engine: `SET SQLITE_PATH_AND_EXE=________`
+        * Save the updated version of the batch file
+    * Edit the SQLite script [SqliteMergeStylx.sql](./SqliteMergeStylx.sql) to provide the fully qualified local path to this folder (`...utilities/style-utilities/merge-stylx-utilities`) 
+        * **IMPORTANT**: the SQLite engine `sqlite3.exe` will *not* recognize backslashes (`\` - MS-DOS paths) so forward slashes (`/` - Unix-style paths) must be used.
+        * Replace all instances (currently 4 instances) of `{FULL-PATH-TO-MERGE-FOLDER-NO-BACKSLASHES}` with the fully qualified local path to this folder: `....utilities/style-utilities/merge-stylx-utilities` being careful to use  forward slashes (`/`) rather than backslashes (`\`)
```

Comment:
Is there any way we can be more specific about where in the .sql file these 4 instances of replacing the local file path  need to be? They are clearly marked with the {FULL-PATH-TO-MERGE-FOLDER-NO-BACKSLASHES} But they are spread out throughout the file.